CONTRIBUTING.md

```diff
@@ -73,27 +73,27 @@ If you have questions about the project, using PyScript, or anything else, pleas
 
 Now that node and npm have both been updated `make setup` should work, and you can continue [setting up your local environment](#setting-up-your-local-environment) without problems (hopefully).
 
-## Creating a Pull Request
+### Building the docs
 
-After you fork PyScript make sure to add a new remote. You can do that using the command:
+To build the documentation locally first make sure you are in the `docs` directory.
+
+You'll need `make` and `conda` installed in your machine. The rest of the environment should be automatically download and created for you once you use the command:
 
 ```
-git remote add remote_nickname remote_url
+make setup
 ```
 
-You should have one remote that corresponds to your local repository and one remote that corresponds the upstream PyScript repository.
-
-We recommend you to create a new local branch and develop your changes in this branch. You can do that by using:
+Use `conda activate $environment_name` to activate your environment.
 
-```
-git checkout -b branch_name #creates new branch
-git checkout branch_name #goes to branch
-```
+To add new information to the documentation make sure you conform with PyScript's code of conduct and with the general principles of Diataxis. Don't worry about reading too much on it, just do your best to keep your contributions on the correct axis.
 
-It's not mandatory but it's advisible to have one issue that is related to your PR, the more information you give and the clearer you can communicate your ideas the faster maintainers will be able to review your changes and merge your PR.
+You need to write your documentation files using [Markedly Structured Text](https://myst-parser.readthedocs.io/en/latest/syntax/optional.html), which is very similar to vanilla Markdown but with some addons to create the documentation infrastructure.
```

Comment:
Delete: `You need to write your documentation files using`
Replace with: `Write using`